rfcs/NNNN-hot-elements.md

```diff
@@ -0,0 +1,77 @@
+---
+Status: Active
+Champions: {{ rictic }}
```

Comment:
remove the `{{ ... }}` - they're supposed to be the placeholder marks :)

Reply:
Ah, I thought they might be a fancy GH markdown feature :)